src/test/java/Interactors/PerformActionSpaceCardTest.java

```diff
@@ -0,0 +1,149 @@
+package Interactors;
+
+import Entities.*;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Objects;
+
+public class PerformActionSpaceCardTest {
+
+    @Test
+    void testGetRandomCard() {
```

Comment:
I think we don't need to test random methods, but good choice here to make every card's type "test".

Reply:
The thing is for the random testing, I'm just showing that it works in that it retrieves some cards from the card list. Of course, there wouldn't be a way to actually test the randomness